### PR TITLE
Add description

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

Adding sentences as suggested by @jrswedlow

### CODE REVIEW

File: history.rst [hunk 1]
```diff
@@ -8,9 +8,11 @@ OMERO version history
 5.4.10 (November 2018)
 ----------------------
 
-This is a bug-fix release.
+This release focuses on improving the Desktop client to ease integration with 3rd party applications e.g. ImageJ/Fiji. The Desktop client is now compatible for offline import using `ome-smuggler <https://github.com/openmicroscopy/omero-ms-queue/>`_.
+Additionally, this release also improves the Web interface when OMERO is opened to the public
```
Comment (mtbc) [inline history.rst]: Think can drop the "additionally" given that there's also "also".

File: history.rst [hunk 2]
```diff
@@ -8,9 +8,11 @@ OMERO version history
 5.4.10 (November 2018)
 ----------------------
 
-This is a bug-fix release.
+This release focuses on improving the Desktop client to ease integration with 3rd party applications e.g. ImageJ/Fiji. The Desktop client is now compatible for offline import using `ome-smuggler <https://github.com/openmicroscopy/omero-ms-queue/>`_.
```
Comment (mtbc) [inline history.rst]: maybe "third-party"

Comment (mtbc) [inline history.rst]: The "The Desktop client is now compatible for offline import using ome-smuggler." is a bit awkward. I'm playing with ideas like,

> The Desktop client can now do offline imports using ome-smuggler.

> The Desktop client now supports ome-smuggler for offline imports.

... or somesuch, not too sure really.

**Comment by jburel** [comment]
@mtbc rephrase as suggested

File: history.rst [hunk 3]
```diff
@@ -8,8 +8,8 @@ OMERO version history
 5.4.10 (November 2018)
 ----------------------
 
-This release focuses on improving the Desktop client to ease integration with 3rd party applications e.g. ImageJ/Fiji. The Desktop client is now compatible for offline import using `ome-smuggler <https://github.com/openmicroscopy/omero-ms-queue/>`_.
-Additionally, this release also improves the Web interface when OMERO is opened to the public
+This release focuses on improving the Desktop client to ease integration with third-party applications e.g. ImageJ/Fiji. The Desktop client can now do offline import using `ome-smuggler <https://github.com/openmicroscopy/omero-ms-queue/>`_.
```
Comment (mtbc) [inline history.rst]: Can we push something to the `omero-ms-queue` `README.md` that more obviously mentions smuggler?

**Review by mtbc — APPROVED** [review]

File: history.rst (hunk 3 above)
Comment (jburel) [reply history.rst]: I think there is a major doc work required in that repo
since all the instructions are not easy to find or build

**Review by mtbc — APPROVED** [review]

**Pull request merged**